app/services/ocr_exporter.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+# This exports OCR files that are in preservation for a provided list of barcodes
+# This is used by researchers who want to get a dump of OCR files from the google books project
+class OCRExporter
+  # @param [String] filename the input file name
+  # @param [String] directory the output directory path
+  def self.export(filename, directory)
+    new(filename, directory).export
+  end
+
+  # @param [String] filename the input file name
+  # @param [String] directory the output directory path
+  def initialize(filename, directory, finder: DruidFinder.new)
+    @filename = filename
+    @directory = directory
+    @finder = finder
+  end
+
+  def export
+    File.open(filename, 'r').each do |line|
+      druid = finder.find_druid(line.chomp)
+      unless druid
+        warn("no druid found for #{line.chomp}")
+        next
+      end
+
+      obj = Dor::Services::Client.object(druid).find
```

Comment:
Skip if file already exists? Will make a retry faster.

Reply:
How do we know what files are expected to exist without loading this?